integration/browserify/runner.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
Might be a good idea to do `set -e` in these bash scripts so that any errors will trigger an early exit.